docs/diagrams/FilePathDiagrams/FilePathSequenceDiagram.puml

```diff
@@ -0,0 +1,110 @@
+@startuml
+!include ../style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":SetPersonFileCommandParser" as SetPersonFileCommandParser LOGIC_COLOR
+participant ":Index" as Index LOGIC_COLOR
+participant ":FilePath" as FilePath LOGIC_COLOR
+participant ":SetPersonFileCommand" as SetPersonFileCommand LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+participant "PersonToEdit:Person" as personToEdit MODEL_COLOR
+participant "EditedPerson:Person" as EditedPerson MODEL_COLOR
+end box
+
+[-> LogicManager : execute("filepath 2 f/C:\Users\...\Tut.pdf")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("filepath 2 f/C:\Users\...\Tut.pdf")
+activate AddressBookParser
+
+create SetPersonFileCommandParser
+AddressBookParser -> SetPersonFileCommandParser
+activate SetPersonFileCommandParser
+
+SetPersonFileCommandParser --> AddressBookParser
+deactivate SetPersonFileCommandParser
+
+AddressBookParser -> SetPersonFileCommandParser : parse("2 f/C:\Users\...\Tut.pdf")
+activate SetPersonFileCommandParser
+
+create Index
+SetPersonFileCommandParser -> Index
+activate Index
+
+Index --> SetPersonFileCommandParser
+deactivate Index
+
+create FilePath
+SetPersonFileCommandParser -> FilePath
+activate FilePath
+
+FilePath --> SetPersonFileCommandParser
+deactivate FilePath
+
+create SetPersonFileCommand
+SetPersonFileCommandParser -> SetPersonFileCommand : (2, "C:\Users\Eugene Tay\Downloads\Tutorial_8_qns.pdf")
+activate SetPersonFileCommand
+
+SetPersonFileCommand --> SetPersonFileCommandParser
+deactivate SetPersonFileCommand
+
+SetPersonFileCommandParser --> AddressBookParser
+deactivate SetPersonFileCommand
+
+AddressBookParser --> LogicManager
+deactivate AddressBookParser
+
+destroy SetPersonFileCommandParser
+
+LogicManager -> SetPersonFileCommand : execute()
+activate SetPersonFileCommand
+
+SetPersonFileCommand -> Model : getFilteredPersonList()
+activate Model
+
+Model --> SetPersonFileCommand
+deactivate Model
+
+SetPersonFileCommand -> SetPersonFileCommand : get(index.getZeroBased())
+activate SetPersonFileCommand
+
+create personToEdit
```

Comment:
Shouldnt be created, should be returned from a get method on the lastShownList

List<Person> lastShownList = model.getFilteredPersonList(); // this part shows where the last shown list comes from
.
.
.
Person personToEdit = lastShownList.get(index.getZeroBased()); // this part shows the getting